shared/common-adapters/search-filter.tsx

```diff
@@ -345,8 +345,6 @@ const styles = Styles.styleSheetCreate(() => ({
   containerMobile: Styles.platformStyles({
     common: {
       paddingBottom: Styles.globalMargins.tiny,
-      paddingLeft: Styles.globalMargins.small,
```

Comment:
prefer to override the style in `profile/search/bar` rather than change all uses of this component

Reply:
could we add this style back?

Reply:
oops, forgot to push that